apps/website/src/server/discord.ts

```diff
@@ -1,52 +1,81 @@
 import { env } from "@/env";
-
 import { triggerOutgoingWebhook } from "@/server/outgoing-webhooks";
 
 export const OUTGOING_WEBHOOK_TYPE_DISCORD_CHANNEL = "discordChannel";
 
 type DiscordWebhookNotificationBody = {
-  // We dont use: files[], embeds, payload_json, attachments, flags, thread_name
   avatar_url: string;
   username: string;
-  content: string;
+  content?: string;
   tts: boolean;
   allowed_mentions: {
     parse?: Array<"everyone" | "roles" | "users">;
     users?: string[];
   };
+  embeds?: Array<{
+    title: string;
+    description: string;
+    color?: number;
+    fields?: Array<{ name: string; value: string; inline?: boolean }>;
+    footer?: { text: string; icon_url?: string };
+    thumbnail?: { url: string };
+    image?: { url: string };
+    author?: { name: string; url?: string; icon_url?: string };
+    timestamp?: Date;
+  }>;
 };
 
 export async function triggerDiscordChannelWebhook({
   webhookUrl,
-  content,
+  contentTitle,
+  contentMessage,
+  contentLink,
+  imageUrl,
   botName: username = env.DISCORD_BOT_NAME,
   expiresAt,
   toEveryone = false,
 }: {
   webhookUrl: string;
-  content: string;
+  contentTitle?: string;
+  contentMessage?: string;
+  contentLink?: string;
+  imageUrl?: string;
   botName?: string;
   expiresAt?: Date;
   toEveryone?: boolean;
 }) {
+  const embed = {
+    title: contentTitle || "Notification",
+    description: contentMessage || "",
```

Comment:
```suggestion
    description: ((contentMessage || "") + (contentLink ? `\n[${contentLink.replace(/^https?:\/\//(www\.)?, "")}](${contentLink})` : "")).trim(),
```
Thoughts on having the link in the content as well for clarity?